### ISSUE TITLE
The steps of installing the application are not completely clear

### ISSUE THREAD

**Issue by YakovMyagkov** [relaypro-open/dog#3]
Hello! I was very interested in your project, I tried to assemble it on my site, but I could not assemble the application from the source code, could you make a more detailed instruction (step by step) how to do it?
I have absolutely no experience in erlang.
It was possible to make an instance through Docker, but I would also like to fully deploy it on the machine as a service

**Comment by dgulinobw** [comment]
Unfortunately, the full Ansible based deployment we use cannot be Open Sourced.

You can run the central portion of dog in Docker - you can expose those services external to the machine via Docker -  then all you have to deploy is the individual dog_agents.  In fact, I think that will be the recommended way of doing things.

A better full deployment HOWTO is still on the TODO.

**Comment by YakovMyagkov** [comment]
Yes, installing docker is easy in terms of a central server, but there are problems with installing the agent, as I wrote earlier in another topic. As I understand correctly, are you offering to deploy the agent on another machine, too, as a docker image, or is it still a full-fledged installation as a service?

**Comment by YakovMyagkov** [comment]
And how i can change url for dog_park from http://dog to http://<ip-address>?
I'm trying change docker-compose, line:    
environment:
      - NGINX_PORT=3030
      - REACT_APP_DOG_API_HOST=http://<my-ip-address>
but it's doesn't work :(

**Comment by YakovMyagkov** [comment]
And is there a delay when transferring settings from Park to Agent ? Because I did change in Profile, but when i'm trying check agent, i see:
root@test-dog-trainer-fw:/opt/dog# docker exec -t -i dog_agent "/usr/sbin/iptables-save"
# Generated by xtables-save v1.8.2 on Tue Sep 27 19:33:57 2022
*filter
:INPUT ACCEPT [0:0]
:FORWARD ACCEPT [0:0]
:OUTPUT ACCEPT [0:0]
COMMIT
# Completed on Tue Sep 27 19:33:57 2022
# Warning: iptables-legacy tables present, use iptables-legacy-save to see them

My profile looks like:
<img width="1246" alt="dog" src="https://user-images.githubusercontent.com/90950462/192619352-dcf8fc2f-bf26-4c0e-bc08-08b2a0007f07.png">

**Comment by dgulinobw** [comment]
1) The delay should be 10-15 seconds tops; more than that and it's not going to ever work.
Did you try `iptables-legacy-save` instead of `iptables-save`?
2) The nginx routing in that docker image requires a hostname to route requests to the proper backend.  You'll need a DNS entry for a central dog server - use that as REACT_APP_DOG_API_HOST=http://yourdog.yourdomain.com
see: https://hub.docker.com/r/jwilder/nginx-proxy

**Comment by YakovMyagkov** [comment]
I used 2 commands, my result looks like:
<img width="957" alt="image" src="https://user-images.githubusercontent.com/90950462/192706588-1e21d040-18e1-488a-b5ea-2245052545ed.png">

**Comment by YakovMyagkov** [comment]
Profile in Dog_trainer it looks like this:
<img width="1256" alt="image" src="https://user-images.githubusercontent.com/90950462/192706894-b7161c7a-e116-4408-9b6c-acb327aaa24b.png">
